tests/test_response.py

```diff
@@ -122,6 +122,24 @@ class RandomException(LambdaFWException):
             "statusCode": 500,
         }
 
+    def test__from_exception__adds_extra_data_to_response_body_if_only_declared(self) -> None:
+        class RandomException(LambdaFWException):
+            error_code = "RAND001"
+
+        response = Response.from_exception(RandomException(extra={"extra": "random"}), "req-id")
+
+        assert response.to_dict() == {
+            "body": (
+                '{"message":"Server got itself in trouble",'
```

Comment:
It would be better to compare `body` separately making a dictionary from that first 😉 

Thanks to that, you will avoid order issues.